src/lib/rest/httpRequestSend.cpp

```diff
@@ -418,7 +427,9 @@ int httpRequestSendWithCurl
   httpHeaderAdd(&headers, "Content-length", headerContentLength, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
 
   // Add the size of the actual payload
```

Comment:
I understand the difference between payloadSize and outgoingMsgSize is that the later is the total size of the message, including also headers and so. It would be useful to explain in a comment... it is not direct (I have had to look to the code a bit in order to realize about the difference).